app/app/settings.py

```diff
@@ -66,6 +66,9 @@
 ALLOWED_HOSTS = env.list('ALLOWED_HOSTS', default=['*'])
 CSRF_TRUSTED_ORIGINS = env.list('CSRF_TRUSTED_ORIGINS', default=['localhost'])
 
+TWILIO_FRIENDLY_NAMES = env.list('TWILIO_FRIENDLY_NAMES', default=['VERIFY'])
```

Comment:
This is for like caller id type info, right? Should we put something about Gitcoin here? And why are we randomly choosing one?

Reply:
> This is for like caller id type info, right? Should we put something about Gitcoin here? And why are we randomly choosing one?

people were worreid about having "GItcoin" go through the SMS towers connected totheir KYC'd phone numbers, especially in authoritarian countries.  so im shuffling the friendly call name in case that helps us obfuscate any pattern that an authoritarian would be trying to pick up. its not perfe[t, but its what we can do